### PR TITLE
Add config for e2e-powervs-periodic

### PR DESCRIPTION
**Pull request by dharaneeshvrd**
Add config job for PowerVS periodic job that runs once a day.

### CODE REVIEW

**Comment by mkumatag** [comment]
/retitle [WIP] - Add config for e2e-powervs

/uncc @csrwng @sjenning

**Comment by dharaneeshvrd** [comment]
@mkumatag Added config for periodic job for PowerVS, please take a look.

**Comment by mkumatag** [comment]
@dharaneeshvrd please run `make jobs` to fix the `ci/prow/generated-config` job

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main__periodics.yaml [hunk 1]
```diff
@@ -139,6 +145,63 @@ tests:
...
+        bin/hypershift install --hypershift-image ${HYPERSHIFT_RELEASE_LATEST}
+
+        # Setting up credentials
+        export IBMCLOUD_CREDENTIALS=/etc/hypershift-pool-powervs-credentials/credentials
+
+        # Complie the e2e tests
+        make e2e
+
+        # Running e2e test
+        bin/test-e2e \
```
Comment (mkumatag) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-main__periodics.yaml:172]: lets also override the control plane image, ref: https://github.com/openshift/release/blob/856e3686acb14f6632959e9253e7295a03bfc203/ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.8.yaml#L99:L106

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main__periodics.yaml (hunk 1 above)
Comment (dharaneeshvrd) [reply ci-operator/config/openshift/hypershift/openshift-hypershift-main__periodics.yaml:177]: Added cpo image as well.

**Review by mkumatag — CHANGES_REQUESTED** [review]
overall lgtm, fix those no newline end of the file errors.

Any idea why that last step is failing for destroying management cluster @csrwng @sjenning?

**Comment by dharaneeshvrd** [comment]
@enxebre is working on something related to use regular aws ci account for hypershift cluster [here](https://github.com/openshift/release/pull/32613/files) as part of that he made some changes to destroy management cluster chain too.
That's why we are seeing these failures I guess.

@enxebre could you please give some insight on this?
And aws-periodic job is also failing on the destroy management cluster step with similar error.  
`{"error": "[failed to describe instances: NoCredentialProviders: no valid providers in chain. Deprecated.\n\tFor verbose messaging see aws.Config.CredentialsChainVerboseErrors, RequestCanceled: request context canceled\ncaused by: context canceled]"}`

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**